mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1625,7 +1625,7 @@
     <!-- Placeholder text shown in the Search String TextField before a user enters text -->
     <string name="search_add_custom_engine_search_string_hint">Search string to use</string>
     <!-- Description text for the Search String TextField. The %s is part of the string -->
-    <string name="search_add_custom_engine_search_string_example">Replace query with “%s”. Example:\nhttps://www.google.com/search?q=%s</string>
+    <string formatted="false" name="search_add_custom_engine_search_string_example">Replace query with “%s”. Example:\nhttps://www.google.com/search?q=%s</string>
```

Comment:
Hi @Amejia481 - I'm wondering why we would use a `formatted="false"` here, since from what I understand it nullifies the placeholders in the string. Thanks!

Reply:
Hi @Delpheen :),
The placeholders `“%s”` are not meant to be replaced, they are just labels in the UI to illustrate how to add a custom search engine. I added `formatted="false"`to let `lint` know that `“%s”` are just labels, and avoid `lint` errors.

![image](https://user-images.githubusercontent.com/773158/126498194-dd050d2d-70d5-492d-ba5b-85e818636315.png)

Reply:
oh :) Ok misunderstood, thanks Arturo!

Reply:
No problem :)